stl/inc/limits

```diff
@@ -1057,19 +1057,22 @@ _NODISCARD int _Checked_x86_x64_countr_zero(const _Ty _Val) noexcept {
     constexpr int _Digits = numeric_limits<_Ty>::digits;
     constexpr _Ty _Max    = (numeric_limits<_Ty>::max)();
 
-#ifndef __AVX2__
-    const bool _Definitely_have_tzcnt = __isa_available >= __ISA_AVAILABLE_AVX2;
-    if (!_Definitely_have_tzcnt && _Val == 0) {
-        return _Digits;
-    }
-#endif // __AVX2__
-
     if _CONSTEXPR_IF (_Digits <= 32) {

```

Comment:
The tests are failing because the special case for 0 is still necessary for the 32-bit case - only for the 8-bit and 16-bit cases can it be skipped.

Reply:
Woops, I was a bit too fast on this - good catch. Will update.

Reply:
Resolved